OpenRA.Mods.RA/Traits/Infiltration/InfiltrateForActor.cs

```diff
@@ -13,18 +13,18 @@
 
 namespace OpenRA.Mods.RA.Traits
 {
-	class InfiltrateForSupportPowerInfo : ITraitInfo
+	class InfiltrateForActorInfo : ITraitInfo
```

Comment:
When traits are changed, mods and maps need to be changed accordingly, otherwise they break. This is done by running the utility commands `--upgrade-mod` and `--upgrade-map`.
Take a look at UpgradeRules.cs, where all "upgrade rules" are located.